src/ocamlorg_frontend/pages/packages.eml

```diff
@@ -115,171 +116,52 @@ Layout.render
     </div>
 </div>
 <div class="bg-white py-20 lg:py-32">
+% let render_package_stat pkg render_extra_stats =
+    <a href=""
```

Comment:
I wonder if for all of them we could just link to `https://v3.ocaml.org/p/<package>` ?

Reply:
Good catch!